src/commands/properties/verify.ts

```diff
@@ -75,10 +72,34 @@ export class PropertyVerifyCommand extends PropertiesUpdateCommand {
         return;
     }
 
-    protected async getProperties(flags: Record<string, any>): Promise<{ [_: string]: string }> {
+    protected async getProperties(flags: Record<string, any>, unikId: string): Promise<{ [_: string]: string }> {
         const properties: { [_: string]: string } = {};
         const verifiedPropertyKey = `${VERIFIED_URL_KEY_PREFIX}${flags["url-name"]}`;
 
+        try {
+            const existingVerifiedUrl = (await this.unsClientWrapper.getUnikProperty(
+                unikId,
+                verifiedPropertyKey,
+                false,
+            )) as ResponseWithChainMeta<string>;
+
+            if (!existingVerifiedUrl.error) {
```

Comment:
And if `existingVerifiedUrl.error` ?

Reply:
@ftreguer  nouvelle proposition Sachant que je ne vois pas comment existingVerifiedUrl.error peut etre setté